pkg/apiserver/diagnose/report_test.go

```diff
@@ -91,6 +191,9 @@ func printRows(t *TableDef) {
 		return
 	}
 
+	fmt.Println(strings.Join(t.Category, " - "))
+	fmt.Println(t.Title)
+	fmt.Println(t.CommentEN)
```

Comment:
Is this necessary?

Reply:
It is just used in the test. Move to the front will know which table has 0 rows.